ci-operator/config/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-main.yaml

```
@@ -56,7 +56,7 @@ tests:
     from: src
 - always_run: false
   as: integration
-  skip_if_only_changed: ^docs/|LICENSE|OWNERS|\.md$|^hack$
+  skip_if_only_changed: ^\.tekton/.*^\.konflux/.*|^docs/|LICENSE|OWNERS|\.md$|^hack$
```

You're missing the `|` in these to have .tekdon and .konflux as separate matches:
```suggestion
  skip_if_only_changed: ^\.tekton/.*|^\.konflux/.*|^docs/|LICENSE|OWNERS|\.md$|^hack$
```